test/bdd/common_steps.go

```diff
@@ -252,7 +252,7 @@ func (d *CommonSteps) jsonPathOfBoolResponseEquals(path, expected string) error
 		return nil
 	}
 
-	return fmt.Errorf("JSON path resolves to [%s] which is not the expected value [%s]", strBool, expected)
+	return fmt.Errorf("JSON path resolves to [%s] which is not the expected value [%s], !!!DEREK response was [%s]", strBool, expected, d.state.getResponse())
```

Comment:
remove this msg

Reply:
Oops, I didn't mean to commit this. I removed it.